app/controllers/projects/new.js

```diff
@@ -1,9 +1,11 @@
 import Controller from '@ember/controller';
```

Comment:
The `projects/new` controller is getting pretty long - we should be trying to group things into meaningful components (like geometry editing)